src/components/Order.js

```diff
@@ -9,7 +9,7 @@ const Order = props => {
     setOrders(orders + 1);
   };
   const subtractOne = () => {
-    setOrders(orders - 1);
+    setOrders(orders === 0 ? orders : orders - 1);
```

Comment:
nice, the order number now never goes below 0, well done! Could you think of a way to approach this so that it only sets state if orders is > 0? At the moment state is getting updated even if orders ===0